cmd/mobile/main.go

```diff
@@ -1,4 +1,4 @@
-// Copyright © 2017 NAME HERE <EMAIL ADDRESS>
+// Copyright Red Hat, Inc., and individual contributors.
```

Comment:
Do we need the license within the start of each file? I think we already have an Apache license on the overall repo. @maleck13

Reply:
not entirely sure either. Does not real harm I guess

Reply:
I guess I'll remove them in that case. It was more so for just keeping consistency

Reply:
No leave them for now

Reply:
Ok